src/Servers/IIS/AspNetCoreModuleV2/RequestHandlerLib/requesthandler_config.h

```diff
@@ -32,7 +32,7 @@
 
 #define MAX_RAPID_FAILS_PER_MINUTE 100
 #define MILLISECONDS_IN_ONE_SECOND 1000
-#define MIN_PORT                   1025
+#define MIN_PORT                   10025
```

Comment:
Is this used somewhere? If not, it should be deleted (and MAX_PORT too).

Reply:
Seems they are not used anywhere. I just deleted them. Thanks!